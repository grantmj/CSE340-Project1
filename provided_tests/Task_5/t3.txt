TASKS
    1 5

POLY 
    F(a) = (1 - 2)(2a)(5a) + (12 - 2)(7);


EXECUTE
    INPUT z;

INPUTS 
    5