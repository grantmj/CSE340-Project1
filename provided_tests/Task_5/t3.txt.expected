POLY - EXPANDED
	F(a) = - 10 a^2 + 70;
